Source/Modules/ocaml.cxx

```diff
@@ -733,7 +733,18 @@ class OCAML:public Language {
 	       "argv = (CAML_VALUE *)malloc( argc * sizeof( CAML_VALUE ) );\n"
 	       "for( i = 0; i < argc; i++ ) {\n" "  argv[i] = caml_list_nth(args,i);\n" "}\n", NIL);
 	Printv(df->code, dispatch, "\n", NIL);
-	Printf(df->code, "caml_failwith(\"No matching function for overloaded '%s'\");\n", iname);
+	Node *sibl = n;
+	while (Getattr(sibl, "sym:previousSibling"))
+	  sibl = Getattr(sibl, "sym:previousSibling");
+	String *protoTypes = NewString("");
```

Comment:
Can you avoid caml case please. The convention is all lower case variable names.

Reply:
I see you copied it from another module, so not to worry. Those are wrong too :(